tools/run_ratbagctl.in

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+#
+# vim: set expandtab shiftwidth=4 tabstop=4:
+#
+# This file is part of libratbag.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice (including the next
+# paragraph) shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+from __future__ import print_function
+import os
+import shutil
+import subprocess
+import sys
+import time
+
+if not os.geteuid() == 0:
+    sys.exit('Script must be run as root')
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+# first copy the policy for the ratbagd daemon to be allowed to run
+shutil.copy(os.path.join(SCRIPT_DIR, 'org.freedesktop.ratbag_devel1.conf'),
+            '/etc/dbus-1/system.d/')
```

Comment:
if we run this through `configure_file` anyway, why not replace the SCRIPT_DIR with the meson builddir?